app/models/tube_rack.rb

```diff
@@ -4,12 +4,15 @@
 # Tubes are linked via the RackedTubes association
 class TubeRack < Labware
   include Barcode::Barcodeable
+  include Asset::Ownership::Unowned
 
   self.sample_partial = 'assets/samples_partials/tube_rack_samples'
 
   has_many :racked_tubes, dependent: :destroy, inverse_of: :tube_rack
   has_many :tubes, through: :racked_tubes
   has_many :contained_samples, through: :tubes, source: :samples
+  # TODO: change to purpose_id
```

Comment:
TODO left in here

Reply:
it's present for other instances in the codebase where plate_purpose_id is used, it refers to that needing a refactor to labware_purpose_id at some point. requires changes to the DB table.
out of scope for this story